### PR TITLE
Language setter

### PR DESCRIPTION
**Pull request by BenjaminCall**
**Description**

Enter a brief description of what you are accomplishing with this Pull Request.

### CODE REVIEW

File: client/js/middleware/serialization/qbank/deserializers/base.js [hunk 1]
```diff
@@ -3,13 +3,19 @@ import { getQbankType }     from '../../../../constants/genus_types';
...
+    languageTypeId = _.last(languages).languageTypeId;
+  }
   return {
     id: item.id,
     type: getQbankType(item.genusTypeId),
     bankId: item.bankId,
     assessmentId: null, // TODO
     name: _.get(item, 'displayName.text'),
-    language: getLanguage(_.get(item, 'displayName.languageTypeId')), // TODO: How does the language even work?
+    language: getLanguage(languageTypeId), // TODO: How does the language even work?
```
Comment (BenHSmith13) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: Do you now know how language even works?

File: client/js/middleware/serialization/qbank/deserializers/base.js [hunk 2]
```diff
@@ -3,13 +3,19 @@ import { getQbankType }     from '../../../../constants/genus_types';
 import { getLanguage }      from '../../../../constants/language_types';
 
 export default function base(item) {
+  const languages = item.question ? item.question.texts : [];
```
Comment (BenHSmith13) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: use _.get, its what all the cool kids are doing

File: client/js/middleware/serialization/qbank/deserializers/base.js (hunk 1 above)
Comment (BenjaminCall) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: haha that was there before. I will kill it

File: client/js/middleware/serialization/qbank/deserializers/base.js [hunk 3]
```diff
@@ -3,13 +3,19 @@ import { getQbankType }     from '../../../../constants/genus_types';
...
+  let languageTypeId = '';
```
Comment (BenHSmith13) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: so do we always send up an empty string?

File: client/js/middleware/serialization/qbank/deserializers/base.js (hunk 2 above)
Comment (BenjaminCall) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: Doesn't work here. _.get return undefinded

File: client/js/middleware/serialization/qbank/deserializers/base.js (hunk 3 above)
Comment (BenjaminCall) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: ... lol I forgot about that. I will fix it

File: client/js/middleware/serialization/qbank/serializers/base.js [hunk 4]
```diff
@@ -1,12 +1,40 @@
 import _                from 'lodash';
 import genusTypes       from '../../../../constants/genus_types';
 import { scrub }        from '../../serializer_utils';
+import { languages, getLanguage } from '../../../../constants/language_types';
 
-export function baseSerializeQuestion(originalItem, newAttributes) {
+function serializeQuestionString(item) {
```
Comment (BenHSmith13) [inline client/js/middleware/serialization/qbank/serializers/base.js]: will this get overridden in any of the other serializers?

File: client/js/middleware/serialization/qbank/deserializers/base.js (hunk 2 above)
Comment (BenHSmith13) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: `_.get(item, 'question.texts', [])`

Comment (BenHSmith13) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: third param is default return

File: client/js/_author/components/assessments/new_item_form.spec.jsx [hunk 5]
```diff
@@ -35,7 +35,7 @@ describe('New Item Form', () => {
     expect(called.type).toBeDefined();
     expect(called.type).toBe('multipleChoice');
     expect(called.name).toBe('');
-    expect(called.language).toBe('english');
+    expect(called.language).toBe('639-2%3AENG%40ISO');
```
Comment (jbasdf) [inline client/js/_author/components/assessments/new_item_form.spec.jsx]: Pull this value from constants ie languages.languageTypeId.english

File: client/js/middleware/serialization/qbank/serializers/base.spec.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import { baseItem } from './base';
 
-describe('baseSerialization', () => {
+fdescribe('baseSerialization', () => {
```
Comment (jbasdf) [inline client/js/middleware/serialization/qbank/serializers/base.spec.js]: Remove fdescribe

File: client/js/middleware/serialization/qbank/deserializers/base.js [hunk 7]
```diff
@@ -3,13 +3,19 @@ import { getQbankType }     from '../../../../constants/genus_types';
 import { getLanguage }      from '../../../../constants/language_types';
 
 export default function base(item) {
+  const languages = _.get(item, 'question.texts', []);
```
Comment (jbasdf) [inline client/js/middleware/serialization/qbank/deserializers/base.js]: We need to change this to show english by default

**Pull request merged**